cdi/embedded/src/main/java/org/infinispan/cdi/util/logging/EmbeddedLog.java

```diff
@@ -21,4 +22,12 @@
    @Message(value = "Configuration for cache '%s' has been defined in cache manager '%s'", id = 17002)
    void cacheConfigurationDefined(String cacheName, EmbeddedCacheManager cacheManager);
 
+   @LogMessage(level = DEBUG)
```

Comment:
Could you please increase it to `INFO`? This information is very helpful when investigating user problem on the forum.